src/configurations/destinations/http/schema.json

```diff
@@ -62,11 +62,11 @@
           "properties": {
             "to": {
               "type": "string",
```

Comment:
what happens if only one of `from` or `to` has been inputted by an user ?

Reply:
we need validation at higher level components or backend

Reply:
As discussed created this task to handle [this](https://linear.app/rudderstack/issue/INT-3246/support-disableemptycolumn-for-mapping-component) scenario. Currently there is no restriction that a column cannot be empty when any other column is filled in that row